docs/en-US/about_Plaster_CreatingAManifest.help.md

```diff
@@ -72,7 +74,19 @@ Data for these parameters can be taken either as parameters to `Invoke-Plaster`,
 - `name`    - The name of the parameter. This is used as the identity to refer to and store the parameter value.
 - `type`    - the type of parameter, currently supported values are `text`, `choice`, `multichoice`, `user-fullname` and `user-email`.
 - `default` - The default value of the parameter, displayed in the UI inside parentheses. This is an optional attribute to assist users of your template. Default values for the `user-fullname` and `user-email` parameter types are retrieved from the user's .gitconfig if no stored value is available.
-- `store`   - Specifies the store type of the value. This is optional, as not all input data should necessarily be stored. The supported types are the same as the available parameter types.
+- `store`   - Specifies the store type of the value. This is optional, as not all input data should necessarily be stored. The supported types are the same as the available parameter types. Stored data is saved to a user profile folder and the filename is a function of the template name and version (`Name-Version-ID.clixml`).
```

Comment:
We should probably mention that these "stored" values will become the new "default" values upon subsequent invocations of that template.  So it is not only a "type" that is important.  The presence of this attribute indicates the template author wants the value the user of the template supplies to become the new default value.  The value of the attribute just determines how that "value" is stored (text or encrypted).